docs/guides/migrate/migration-guide.md

```diff
@@ -767,7 +767,7 @@ In ZIO 2.x, we added the `Unsafe` data type to help developers to differentiate
 
 ```scala
 object Unsafe {
-  def unsafe[A](f: Unsafe => A): A = ???
+  def unsafely[A](f: Unsafe => A): A = ???
```

Comment:
You only renamed it in the docs and in a comment

Reply:
Ah I originally renamed it and didn't change these back. Fixed now.